Src/WindowsServer/WindowsServer.Shared/FirstChanceExceptionStatisticsTelemetryModule.cs

```diff
@@ -206,9 +214,13 @@ private void TrackStatistics(string type, string operation, string method, bool
         {
             var dimensions = new Dictionary<string, string>();
             dimensions.Add("type", this.GetDimCappedString(type, this.typeValues));
-            dimensions.Add("method", this.GetDimCappedString(method, this.methodValues));
+            dimensions.Add("Failed method", this.GetDimCappedString(method, this.methodValues));
```

Comment:
Minor: Does "." notation in Analytics query support spaces in the property names? If not - we might be limiting query time choices by adding a space.